incubator/spring-cloud-data-flow/templates/server-config-rabbit.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "scdf.fullname" . }}-server
+  labels:
+    app: {{ template "name" . }}
+    component: {{ template "name" . }}-server
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+data:
+  application.yaml: |-
+    security:
+      basic:
+        enabled: true
+        realm: Spring Cloud Data Flow
+    spring:
+      cloud:
+        dataflow:
+          security:
+            authentication:
+              file:
+                enabled: true
+                users:
```

Comment:
I thought I did - I added a `templates/secrets.yaml` file and use Spring Boot property reference place holders instead of the template Values placeholders for the passwords